CCContinuousGlucose/Classes/ContinuousGlucoseAnnunciation.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ContinuousGlucoseAnnunciation.swift
+//  Pods
+//
+//  Created by Kevin Tallevi on 4/19/17.
+//
+//
+
+import Foundation
+import CCToolbox
+
+public class ContinuousGlucoseAnnunciation : NSObject {
+    public var sessionStopped: Bool?
+    public var deviceBatteryLow: Bool?
+    public var sensorTypeIncorrectForDevice: Bool?
+    public var sensorMalfunction: Bool?
+    public var deviceSpecificAlert: Bool?
+    public var generalDeviceFaultHasOccurredInTheSensor: Bool?
+    public var timeSynchronizationBetweenSensorAndCollectorRequired: Bool?
+    public var calibrationNotAllowed: Bool?
+    public var calibrationRecommended: Bool?
+    public var calibrationRequired: Bool?
+    public var sensorTemperatureTooHighForValidTestResultAtTimeOfMeasurement: Bool?
+    public var sensorTemperatureTooLowForValidTestResultAtTimeOfMeasurement: Bool?
+    public var sensorResultLowerThanThePatientLowLevel: Bool?
+    public var sensorResultHigherThanThePatientHighLevel: Bool?
+    public var sensorResultLowerThanTheHypoLevel: Bool?
+    public var sensorResultHigherThanTheHyperLevel: Bool?
+    public var sensorRateOfDecreaseExceeded: Bool?
+    public var sensorRateOfIncreaseExceeded: Bool?
+    public var sensorResultLowerThanTheDeviceCanProcess: Bool?
+    public var sensorResultHigherThanTheDeviceCanProcess: Bool?
+    
+    private var sessionStoppedBit = 0
```

Comment:
These are variables that are set to true/false within the class, so have to be var's